src/Classes/Controller/AbstractDocumentFormController.php

```diff
@@ -215,7 +213,7 @@ public function createAction(DocumentForm $newDocumentForm)
         }
 
         // xml data fields are limited to 64 KB
-        if (strlen($newDocument->getXmlData()) >= 64 * 1024 || strlen($newDocument->getSlubInfoData() >= 64 * 1024)) {
+        if (strlen($newDocument->getXmlData()) >= 64 * 1024) {
```

Comment:
Should not this be configurable or specified as a constant? Otherwise this is a magic number, hidden in the code, only explained by a comment.

Reply:
The magic number has been replaced by a constant.